content/03-reference/03-more/02-telemetry.mdx

```diff
@@ -26,7 +26,10 @@ Data is collected in two scenarios that are described below.
 
 ### Usage data
 
-Invocations of the `prisma` CLI sends information to the telemetry server at https://checkpoint.prisma.io. Note that this is only happening at most every 48 hours (i.e., the sending the data to the telemetry server gets paused for 48 hours after any invocation).
+Invocations of the `prisma` CLI and general usage of Studio results in data being sent to the telemetry server at https://checkpoint.prisma.io. Note that:
+
+* Information is only sent every 48 hours (sending is paused for 48 hours after any invocation)
```

Comment:
Don't think this is correct: The duration is now 12 hours @2color @matthewmueller?

Reply:
Yes. It's 12 hours.

Reply:
Fixed - thanks!